lib/output/utils.js

```diff
@@ -89,27 +99,42 @@ const namedDelete = Symbol("named property delete");
 module.exports = exports = {
   isObject,
   hasOwn,
+  /** @type {typeof wrapperSymbol} */
```

Comment:
This&nbsp;wouldn’t be&nbsp;necessary if&nbsp;we&nbsp;used the&nbsp;following&nbsp;instead:
```js
const wrapperSymbol = Symbol("wrapper");
exports.wrapperSymbol = wrapperSymbol;
```

<sup><sup><sub>This&nbsp;feels more&nbsp;like a&nbsp;quirk of&nbsp;**Salsa**&nbsp;though (**TypeScript**’s&nbsp;**JavaScript** JSDoc&nbsp;IntelliSense&nbsp;engine).</sub></sup></sup>